rotkehlchen/exchanges/iconomi.py

```diff
@@ -0,0 +1,260 @@
+import base64
+import hashlib
+import hmac
+import logging
+import time
+from json.decoder import JSONDecodeError
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple
+from urllib.parse import urlencode
+
+from requests.exceptions import ReadTimeout
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.errors import RemoteError, UnknownAsset
+from rotkehlchen.exchanges.data_structures import Location, Price, Trade, TradePair, TradeType
+from rotkehlchen.exchanges.exchange import ExchangeInterface
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.typing import ApiKey, ApiSecret, Timestamp
+from rotkehlchen.user_messages import MessagesAggregator
+from rotkehlchen.utils.serialization import rlk_jsonloads
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+SUPPORTED_FUND_TICKERS = ('BLX',)
```

Comment:
So only one asset is supported for iconomi? I have not really used it. What is a fund ticker?

Reply:
I removed support for ICONOMI funds/strategies. Currently only single assets are supported.

BLX worked in previous versions of Rotki because it was tokenized as an ERC20 as well. But since Rotki removed support for BLX, I removed it from the importer as well.

Reply:
In ICONOMI all funds/strategies had tickers and they wanted to tokenize them as well. But I think they no longer planning to do that.